CHANGELOG.md

```diff
@@ -4,6 +4,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.0.0-rc.1] - 2022-03-17
+
+### Added
+
+- The CQL2 Accent and Case-insensitive Comparison 
+    (`http://www.opengis.net/spec/cql2/1.0/conf/accent-case-insensitive-comparison`) conformance class
+    adds the ACCENTI and CASEI functions for case-insensitive comparison. These replace the UPPER and
+    LOWER psuedo-functions that were previously part of the Advanced Comparison Operators class.
+
+### Changed
+
+- Query Extension is no longer deprecated.
+- Children conformance class now requires the /children endpoint includes all child catalogs or collections
+  referenced via `child` link relations from the Landing Page
+- Clarified behavior of Transaction Extension endpoints:
+  - PUT and PATCH of a body that changes the `collection` or `id` is disallowed.
+  - POST, PUT, and PATCH do not need to include the `collection` attribute, as it should be derived from the URL.
+  - POST and PUT can be used with a body that is at least a GeoJSON Feature, but does not have to be an Item, but for which 
+    the server can derive a valid Item, e.g., by populating the id and collection fields or adding links
```

Comment:
does this mean the geojson feature has to have an id field?

Reply:
good question -- I had thought geojson feature required id, but now I see it's a "should". So maybe we need to clarify that behavior -- maybe just say that it's up to the implementer if they want to autogenerate IDs.

Reply:
Decided the answer to this is "yes". The openapi spec already defined `id` as a required field.